lib/symengine.rb

```diff
@@ -33,6 +33,29 @@ def Function(n)
     def evalf(operand, prec=53, real=false)
         return _evalf(operand, prec, real)
     end
+    def evalf_dirichlet_eta(exp)
+        SymEngine::evalf(SymEngine::dirichlet_eta(exp))
+    end
+    def evalf_zeta(exp)
+        SymEngine::evalf(SymEngine::zeta(exp))
+    end
+    def lambdify_code(exp)
+      sym_map = exp.free_symbols.map { |sym| sym.to_s}.sort.join(",")
```

Comment:
The above solution didn't work out properly. The order of `to_s` function is also similar.. it's not in line with the original symbol order.. so we have to limit it to giving a set of symbols